### DISCUSSION TITLE
Linux arm64 bottles

### DISCUSSION THREAD

**Discussion by leighmcculloch** [Homebrew/discussions#3939] [Linux] [answered]
Is it possible to build Linux arm64 bottles as part of the general CI build process?

**Comment by carlocab** [comment, marked as answer]
Not in https://github.com/Homebrew/homebrew-core. At least, not until GitHub provides Linux ARM64 runners. However, you might be interested in https://github.com/gromgit/homebrew-core-aarch64_linux. Note: this is a third-party repository that is not endorsed or promoted in any way by the Homebrew organisation.
